_posts/2017/2017-03-28-epages-hackathon-2017.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "ePages Hackathon 2017"
+date: "2017-03-28 06:39:20"
+image: blog-header/blue-sky.jpg
+categories: events
+authors: ["Rose"]
+---
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+As every year in spring, last week we organized our third hackathon at ePages.
+This year's hackathon has been all about coding for and further developing our very latest backend architecture ePages BEYOND, adding value to our current core shop software ePages 6 as well as automating internal processes.
+
+### Getting prepared
+
+Just as last year, our Scrum Masters did an outstanding job organizing and ensuring a smooth flow of the two-day hackathon.
+They divided up teams that usually don't work with each other on a regular basis, they agreed upon final topics, booked meeting rooms for the teams and most importantly ordered delicious snacks, drinks and food {% emoji wink %}.
+
+A day before the hackathon, our colleagues from Jena and Barcelona arrived in Hamburg to join the hacking fun at our headquarter.
+
+### Let's get started!
+
+The first day started off with a surprisingly blue sky and a vitalizing breakfast to get all developers fueled and ready to hack away.
```

Comment:
Add an additional sentence:

The atmosphere was very relaxed and we had lot's of time to chat with colleagues we had not seen for some time.